packages/devtools-launchpad/package.json

```diff
@@ -14,6 +14,7 @@
   "scripts": {
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
+    "license-check": "../devtools-license-check/bin/devtools-license-check",
```

Comment:
Does that mean we'd have to do a release each time we need to change the license checker, even to get it working for devtools-core?